osu.Game.Rulesets.Osu.Tests/Editor/TestSceneSliderPlacementBlueprint.cs

```diff
@@ -61,6 +61,21 @@ public void TestPlaceWithMouseMovement()
             assertControlPointType(0, PathType.Linear);
         }
 
+        [Test]
+        public void TestPlaceWithMouseMovementOutsidePlayfield()
+        {
+            addMovementStep(new Vector2(200));
+            addClickStep(MouseButton.Left);
+
+            addMovementStep(new Vector2(1400, 200));
```

Comment:
This test implicitly relies on screen width to fail. When I checked the test out to see it fail, it initially didn't, because my main monitor is ultrawide and the cursor didn't go out of bounds.

Not sure it's worth fixing, but while we're here I figured I'd bring up? Probably easiest to just make number bigger..

Reply:
I've fixed in a slightly different way, see what you think.